lib/validate.js

```diff
@@ -1,22 +1,38 @@
 var merge = require('mout/object/merge')
 var semver = require('semver')
 var subjectExceptions = /^(fixup|squash)!|^\[[^\]]+\]:/
+var styles = Object.freeze({ JQUERY: 0, ANGULAR: 1 })
```

Comment:
Let's replace this with plain strings, so that the option becomes `style: "angular"`, which can be specified in `package.json`.

So drop this `styles` object entirely and replace the relevant check with just `style === "angular"`.